evennia/utils/evmenu.py

```diff
@@ -64,10 +68,37 @@ def another_node(caller, input_string, **kwargs):
 The return values must be given in the above order, but each can be
 returned as None as well. If the options are returned as None, the
 menu is immediately exited and the default "look" command is called.
-
-    text (str, tuple or None): Text shown at this node. If a tuple, the
-        second element in the tuple is a help text to display at this
-        node when the user enters the menu help command there.
+    text (str, tuple, dict or None): Text shown at this node. If a dict
+        it can also contain various formatting instructions. In such a 
+        case the followng keys are already supported:        
+            - 'text' (str): The text of the node.
+            - 'help' (str): The help text of the node.
+            - 'footer' (str): Footer text for the node.
+            - 'format' (str): String dictating the layout of the node if
+                the default layout is not being used. It currently
+                supports two options:
+                    - 'suppress': The node is not drawn at all
+                    - 'scroll': The node is framed inside the ANSI image
+                        of a scroll.
+            - 'optionformat' (dict): A dictionary containing information 
+                for the formating of the options table. Current options 
+                are:
+                    - 'hidekey' (list): A list of keys. The options for 
+                        those keys will not be listed in the options 
+                        table though the node will accept them as input.
+                    - 'hidedesc' (list): As hidekey but using the desc 
+                        of the options instead.
+                    - 'movekeys' (list): A list of keys. The options for 
+                        those keys will be put into a separate list 
+                        after the options table.
+                    - 'movedesc' (list): As movekeys but using the desc
+                        of the options instead.
+                    - 'rows' (int): Will attempt to make the options 
+                        table the specified number of rows.            
```

Comment:
This is a good feature I think.

Reply:
Thanks. Being able to control this on a node-by-node basis was one of the primary motivators for this.